packages/authentication/docs/setup-email-link.md

```diff
@@ -21,6 +21,24 @@
 		```
 
 		`[DOMAIN_NAME]` must be replaced with your domain name of the link.
+	
+	2. If the above doesnt work try the following:
```

Comment:
There must be a reason why it doesn't work. Please debug it. The setup instructions must be clear so that you know when to use which variant.